README.md

```diff
@@ -38,13 +38,51 @@ The Linera protocol repository is broken down into the following crates and subd
 The following script can be run with `cargo test`.
 
 ```bash
+storage="ROCKSDB"
+
+if [ $# -eq 1 ]
+then
+    if [ "$1" == 'DYNAMODB' ]
+    then
+        storage="DYNAMODB"
+    fi
+    if [ "$1" == 'SCYLLADB' ]
+    then
+        storage="SCYLLADB"
+    fi
+fi
+
 # For debug builds:
-cargo build && cd target/debug
+if [ "$storage" = "ROCKSDB" ]
+then
+    cargo build && cd target/debug
+elif [ "$storage" = "DYNAMODB" ]
+then
+    cargo build --features aws && cd target/debug
+elif [ "$storage" = "SCYLLADB" ]
+then
+    cargo build --features scylladb && cd target/debug
+fi
+
 # For release builds:
 # cargo build --release && cd target/release
 
 # Clean up data files
 rm -rf *.json *.txt *.db
+rm -rf linera.db
+if [ "$storage" = "ROCKSDB" ]
+then
+    rm -rf server_?_?.db
+elif [ "$storage" = "DYNAMODB" ]
+then
+    ./linera-db --command delete_all --storage dynamodb:table:localstack
```

Comment:
Let's use `structopt`'s subcommands (see other tools) and remove `--command` here.

Reply:
Ok, done.